tests/functionals/classes/asserters/generator.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace mageekguy\atoum\tests\functionals\asserters;
+
+use
+	mageekguy\atoum
+;
+
+require_once __DIR__ . '/../../runner.php';
+
+class generator extends atoum\tests\functionals\test\functional
+{
+	/**
+	 * @php >= 7.0
+	 */
+	public function testUsage()
+	{
+		if (version_compare(PHP_VERSION, '7.0') >= 0) {
+			$generator = eval(<<<'PHP'
```

Comment:
yes, in php 5.6 there is an error when the generator returns something, even with the version_compare. The eval avoids that.